README.md

```diff
@@ -61,6 +59,10 @@ puts GithubService.new.repositories('alebian')
 ```
 
 Or, even easier, run `rails generate wor:requests:service NAME` in your Rails root
+#### options in service generator
+- module
+
+  `rails generate wor:requests:service NAME --module MODULE_NAME`
```

Comment:
:thinking: I would use the generators as an independent paragraph, and `Usage` paragraph should be `Examples`, What do you think about this?

<details><summary>Instead of this</summary>
<p>

## Usage

To write your first Service using Wor-requests you can write something like this:

```ruby
require 'wor/requests'

class GithubService < Wor::Requests::Base
  def repositories(username)
    get(
      attempting_to: "get repositories of #{username}",
      path: "/users/#{username}/repos"
    )
  rescue Wor::Requests::RequestError => e
    puts e.message
  end

  protected

  def base_url
    'https://api.github.com'
  end
end

puts GithubService.new.repositories('alebian')
```

Or, even easier, run `rails generate wor:requests:service NAME` in your Rails root
#### options in service generator
- module

  `rails generate wor:requests:service NAME --module MODULE_NAME`


</p>
</details>


--------------------------------

<details><summary>I prefer to use something like this</summary>
<p>

## Generators
```bash
rails generate wor:requests:service NAME # in your Rails root
```
### Generators options
#### module
Specifying the module name as:
```bash
rails generate wor:requests:service NAME --module MODULE_NAME
```
We can create a service with an inner class NAME, and external module called MODULE_NAME
```rb
module ModuleName
  class NameService < Wor::Requests::Base
    # Your code here
  end
end
```

## Service example

To write your first Service using Wor-requests you can write something like this:

```ruby
require 'wor/requests'

class GithubService < Wor::Requests::Base
  def repositories(username)
    get(
      attempting_to: "get repositories of #{username}",
      path: "/users/#{username}/repos"
    )
  rescue Wor::Requests::RequestError => e
    puts e.message
  end

  protected

  def base_url
    'https://api.github.com'
  end
end

puts GithubService.new.repositories('wolox')
```


</p>
</details>